api/v1alpha1/perconaservermysql_types.go

```diff
@@ -124,6 +125,30 @@ type PMMSpec struct {
 	RuntimeClassName         *string                     `json:"runtimeClassName,omitempty"`
 }
 
+type BackupSpec struct {
+	Image                    string                        `json:"image,omitempty"`
+	ImagePullSecrets         []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+	ImagePullPolicy          corev1.PullPolicy             `json:"imagePullPolicy,omitempty"`
+	ServiceAccountName       string                        `json:"serviceAccountName,omitempty"`
+	ContainerSecurityContext *corev1.SecurityContext       `json:"containerSecurityContext,omitempty"`
+	Resources                corev1.ResourceRequirements   `json:"resources,omitempty"`
+	Storages                 map[string]*BackupStorageSpec `json:"storages,omitempty"`
+}
+
+type BackupStorageType string
+
+const (
+	BackupStorageFilesystem BackupStorageType = "filesystem"
+	BackupStorageS3         BackupStorageType = "s3"
+	BackupStorageGCS        BackupStorageType = "gcs"
+	BackupStorageAzure      BackupStorageType = "azure"
+)
+
+type BackupStorageSpec struct {
```

Comment:
please see https://github.com/percona/percona-xtradb-cluster-operator/blob/main/pkg/apis/pxc/v1/pxc_types.go#L433-L449 and all needed features like Annotations, Labels and so on

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/60/commits/1789ad956beb6221d12ee16008737653097ed68f